sublime_syntaxes/Assembly x86.sublime-syntax

```diff
@@ -0,0 +1,67 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/3/syntax.html
+name: Assembly x86 (NASM)
+file_extensions:
+  - asm
+  - inc
```

Comment:
Hmm… So the best solution would be to send an upstream PR that changes the tmLanguage file?

I think the `.sublime-syntax` files would suffice since most syntax files are only updated infrequently. It would also make the build process easier by reducing the number of submodules. On the other hand, we might miss some syntax updates…